scripts/sign.py

```diff
@@ -4,55 +4,99 @@
 #
 # SPDX-License-Identifier: BSD-2-Clause
 
-
 def uuid_parse(s):
     from uuid import UUID
     return UUID(s)
 
-
 def int_parse(str):
     return int(str, 0)
 
 
 def get_args():
+    import sys
+    print(sys.argv)
```

Comment:
looks like debug stuff.
Remove or use `logger` for the trace.

Reply:
Yes, debug leftover. Will remove. Just wanted a quick feedback before I finally polish the code/comments (typos etc.)